zeppelin-web-vue/src/components/Notebook/Controls.vue

```diff
@@ -52,52 +56,73 @@
       <a
         href="javascript: void(0);"
         @click="showDeleteConfirm"
+        v-if="!isDeleted"
       >
         <a-tooltip placement="top">
           <template slot="title">
-            <span>Delete</span>
+            <span>Move to Recycle Bin</span>
```

Comment:
`Recycle Bin` is very windows specific, should we call it trash?

Reply:
Renamed Recycle Bin to Trash.